$OpenBSD$

set our chroot install path
we do this later to not interfere with creating the proper config file paths

Index: web/CMakeLists.txt
--- web/CMakeLists.txt.orig
+++ web/CMakeLists.txt
@@ -9,6 +9,7 @@ add_subdirectory(tools/mootools)
 configure_file(includes/config.php.in "${CMAKE_CURRENT_BINARY_DIR}/includes/config.php" @ONLY)
 
 # Install the web files
+set(ZM_WEBDIR "/var/www/htdocs/zm")
 install(DIRECTORY vendor api ajax css fonts graphics includes js lang skins tools views DESTINATION "${ZM_WEBDIR}" PATTERN "*.in" EXCLUDE PATTERN "*Make*" EXCLUDE PATTERN "*cmake*" EXCLUDE)
 install(FILES index.php robots.txt DESTINATION "${ZM_WEBDIR}")
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/includes/config.php" DESTINATION "${ZM_WEBDIR}/includes")
